scss/atoms/modals/_modals.scss

```diff
@@ -0,0 +1,33 @@
+/*
+* ==========================================================================
+* Modals
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
+	"mixins";
+
+.modal {
+	@include modal();
+
+	.modal-header {
+		@include modal-header();
+		h4 {
```

Comment:
In general we want to avoid cascading as well. Have modal-title so they can use whatever element they need semantically, and remove modal-header cascading from inside modal.

Sent from my iPhone

> On May 27, 2016, at 4:43 PM, Chris Rebert <notifications@github.com> wrote:
> 
> In scss/atoms/modals/_modals.scss:
> 
> > +* ==========================================================================
> > +* Modals
> > +* ==========================================================================
> > +*/
> > +
> > +@import
> > +	"dist/chassis",
> > +	"functions",
> > +	"mixins";
> > +
> > +.modal {
> > +	@include modal();
> > +
> > +	.modal-header {
> > +		@include modal-header();
> > +		h4 {
> What about a .modal-title class then?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>